src/cmd/prune.ts

```diff
@@ -0,0 +1,9 @@
+import prune from '../api/prune'
+import {PublicInstallationOptions} from '../api/install'
+
+export default (input: string[], opts: PublicInstallationOptions) => {
+  if (input && input.length) {
+    return prune(input, opts)
+  }
+  return prune(opts)
```

Comment:
> But opts seems more natural as a last parameter

in fp it is not.

> Maybe lets create two separate functions?

i dont know

> Or lets move input to a property in opts?

seems like bad idea, two separate responsibilities merged in one object